task/buildah-remote-oci-ta/0.2/buildah-remote-oci-ta.yaml

```diff
@@ -447,13 +447,16 @@ spec:
         done < <(find $ADDITIONAL_SECRET_TMP -maxdepth 1 -type f -exec basename {} \;)
       fi
 
-      unshare -Uf $UNSHARE_ARGS --keep-caps -r --map-users 1,1,65536 --map-groups 1,1,65536 -w ${SOURCE_CODE_DIR}/$CONTEXT -- buildah build \
-        $VOLUME_MOUNTS \
-        "${BUILDAH_ARGS[@]}" \
-        "${LABELS[@]}" \
-        --tls-verify=$TLSVERIFY --no-cache \
-        --ulimit nofile=4096:4096 \
-        -f "$dockerfile_path" -t $IMAGE .
+      COMMAND="ip link set lo up;
```

Comment:
You needed to change the quay.io/konflux-ci/buildah image to include the necessary package. Does this package already exist on all of the remote VMs?

Reply:
I should not need it on the VM. It only needs to be in $BUILDER_IMAGE which again defaults to quay.io/konflux-ci/buildah-task  ([reference](https://github.com/konflux-ci/build-definitions/blob/109199efc4d9823d875a376df06929dd50fdee38/task/buildah-remote-oci-ta/0.2/buildah-remote-oci-ta.yaml#L193C5-L194C46))

Reply:
ah, right.